atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClient.java

```diff
@@ -142,23 +152,24 @@ String system_add_keyspace(KsDef ks_def)
     String system_update_keyspace(KsDef ks_def)
             throws InvalidRequestException, SchemaDisagreementException, TException;
 
-    String system_add_column_family(CfDef cf_def)
```

Comment:
unused?

Reply:
yeah, everything builds without it? See PR comment

Reply:
But could be that internal products use this? @felixdesouza how can we check that?

Reply:
I usually just open it up, it's unused inside big internal product.

Reply:
Sweet, so should be fine to nuke it.